include/exadg/solvers_and_preconditioners/preconditioners/preconditioner_amg.h

```diff
@@ -2,7 +2,7 @@
  *
  *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
  *
- *  Copyright (C) 2021 by the ExaDG authors
+ *  Copyright (C) 2021 by the ExaDG authors and Marco Feder
```

Comment:
I think we should not introduce this style. If you want, you can add this name to the documentation of the particular function that was inspired by this developer. However, a link to deal.II code or some other publicly available repo would be better in my opinion.

Reply:
I agree. Also, the code has evolved over many places if I recall the history. There are several pieces originally from deal.II, that were later extended to manual null spaces in 2015/16 in https://github.com/kronbichler/adaflo/blob/6329eeec998ce7f4d527f4fed96b9043e9ca2106/source/navier_stokes_preconditioner.cc#L474-L532 and additional evolution towards structural problems by different people. But I would definitely support listing contributions by Marco in the authors page.

Reply:
Since the stuff he and Peter provided is now in that unknown dealii branch that has DoFTools::extract_level_elasticity_modes() and  DoFTools::extract_elasticity_modes(), I think his contributions are then to be valued by deal.II, does that make sense?